packages/twenty-front/src/modules/views/hooks/__tests__/useViewBar.test.tsx

```diff
@@ -0,0 +1,831 @@
+import { act } from 'react-dom/test-utils';
```

Comment:
this test file is too long and should be broken in smaller files
Intersting work though

Reply:
Alright